src/osd/OSD.h

```diff
@@ -1194,6 +1194,7 @@ class OSD : public Dispatcher,
 
   // -- superblock --
   OSDSuperblock superblock;
+  ceph::coarse_mono_clock::time_point last_trim_maps = ceph::coarse_mono_clock::zero();
```

Comment:
Line length exceeds 80, please separate.

Reply:
what's the point of assigning `ceph::coarse_mono_clock::zero()` to it. the value of the default constructor should be good enough, i think.